discord/client.py

```diff
@@ -197,6 +198,32 @@ class Client:
         To enable these events, this must be set to ``True``. Defaults to ``False``.
 
         .. versionadded:: 2.0
+    bucket_storage_cls: type[:class:`.BucketStorageProtocol`]
+        The class to use for storing rate limit buckets given by Discord.
+
+        .. versionadded:: 2.5
+    concurrency: :class:`int`
```

Comment:
I would rename this and the `per` parameters to include ratelimit in the name, to be more descriptive. Just like `rate_limit_timeout` (although I would personally rename that to `ratelimit_timeout`).